packages/reakit/src/Tooltip/Tooltip.tsx

```diff
@@ -20,7 +20,6 @@ export type TooltipOptions = DisclosureContentOptions &
   > & {
     /**
      * Whether or not the dialog should be rendered within `Portal`.
```

Comment:
Thanks a lot @mattrothenberg! I think we should also replace `dialog` with `tooltip`.
```suggestion
     * Whether or not the tooltip should be rendered within `Portal`.
```